### ISSUE TITLE
6.4.0 Rolling Release 

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#9994]
### Prerequisites

* [x] DEV: release reva v.2.24.1
* [x] DEV: bump reva to v.2.24.1 https://github.com/owncloud/ocis/pull/10028
* [x] DEV: bump web to v.10.2.0 https://github.com/owncloud/web/releases/tag/v10.2.0

## QA Phase
* [x] QA: Performance test 
* [x] QA: Compatibility testing between `owncloud/ocis-rolling:6.3.0` and https://hub.docker.com/layers/owncloud/ocis-rolling/master/images/sha256-a67ad8c515971c8224f4271640e25fa8bf7242b2be1e291fdacf34d28962867c?context=explore (similar with owncloud/ocis-rolling:6.4.0)
* [x] QA: e2e with different deployment: https://github.com/owncloud/web/pull/11570

### After QA Phase
* [x] DEV: Create branch `release-x.x.x` https://github.com/owncloud/ocis/pull/10037
  * [x] DEV: bump ocis version in necessary files
  * [x] DEV: `ocis-pkg/version/version.go`
  * [x] DEV: `sonar-project.properties`
  * [x] DEV: released deployment versions
  * [x] DEV: prepare changelog folder in `changelog/x.x.x_???`
  * [x] DEV: add release to https://owncloud.dev/ocis/release_roadmap
* [x] DEV: Create final signed tag
* [x] DEV: Check successful CI run on `vx.y.z` tag / BLOCKING for all further activity
* [x] Merge release notes https://github.com/owncloud/docs-main/pull/73
* [x] roll out release on https://infinite.owncloud.com/
* [x] roll out rolling release on https://owncloud.dev/ocis/deployment/continuous_deployment/ 

****

**Comment by ScharfViktor** [comment]
### K6 test:
✅️ see result here https://grafana.k6.infra.owncloud.works/?orgId=1&refresh=15m&var-test_id=1dcb6ffd-5b8b-4d22-b3f2-c3f8ea26e070

```
     ✓ authn -> logonResponse - status
     ✓ authn -> authorizeResponse - status
     ✓ authn -> accessTokenResponse - status
     ✓ client -> user.findUser - status
     ✓ client -> role.getMyDrives - status
     ✓ client -> resource.getResourceProperties - status
     ✓ client -> application.createDrive - status
     ✓ client -> drive.deactivateDrive - status
     ✓ client -> resource.createResource - status
     ✓ client -> drive.deleteDrive - status
     ✓ client -> tag.addTagToResource - status
     ✓ client -> resource.downloadResource - status
     ✓ client -> group.findGroup - status
     ✓ client -> tag.getTagForResource - status
     ✓ test -> resource.getTags - name - match
     ✓ client -> tag.removeTagToResource - status
     ✓ client -> share.createShareInvitation - status
     ✓ client -> resource.uploadResource - status
     ✓ client -> share.deleteShareInvitation - status
     ✓ client -> resource.moveResource - status
     ✓ client -> resource.deleteResource - status

     checks.........................: 100.00% ✓ 52368     ✗ 0    
     data_received..................: 24 GB   18 MB/s
     data_sent......................: 16 GB   12 MB/s
     http_req_blocked...............: avg=55.87µs  p(95)=7.37µs   p(99)=11.07µs  count=51095
     http_req_connecting............: avg=4.5µs    p(95)=0s       p(99)=0s       count=51095
     http_req_duration..............: avg=119.44ms p(95)=300.69ms p(99)=705.91ms count=51095
       { expected_response:true }...: avg=119.44ms p(95)=300.69ms p(99)=705.91ms count=51095
     http_req_failed................: 0.00%   ✓ 0         ✗ 51095
     http_req_receiving.............: avg=9.17ms   p(95)=486.13µs p(99)=352.69ms count=51095
     http_req_sending...............: avg=4.46ms   p(95)=40.85µs  p(99)=1.14ms   count=51095
     http_req_tls_handshaking.......: avg=45.48µs  p(95)=0s       p(99)=0s       count=51095
     http_req_waiting...............: avg=105.79ms p(95)=265.23ms p(99)=412.26ms count=51095
     http_reqs......................: 51095   38.392374/s
     iteration_duration.............: avg=11.7s    p(95)=19.62s   p(99)=23.97s   count=28542
     iterations.....................: 28542   21.44623/s
     vus............................: 1       min=0       max=300
     vus_max........................: 300     min=50      max=300
```

**Comment by ScharfViktor** [comment]
### QA: e2e with different deployment 
- ✅️ all tests pass except of tests/e2e/cucumber/features/journeys/kindergarten.feature:21 in keycloak (not related release)

**Comment by ScharfViktor** [comment]
### Compatibility test between `owncloud/ocis:6.2.0` and `owncloud/ocis-rolling:master (a2a786a5e52c503614964ac9b575c7b08ff8b762728dd0acad0ec06a4322ccd5 similar with ocis:6.3.0)`

shares:
- [x] public share (with an expiration date)
- [x] share with user as role editor (with an expiration date)
- [x] share with user as role viewer
- [x] share with group "Users"

project space:
- [x] uploaded folder & files
- [x] trashed files
- [x] share space with user
- [x] public share space

files:
- [x] uploaded folder & files
- [x] version of files
- [x] trashed files
- [x] GDPR export

users
- [x] non demo user with role admin
- [x] non demo user with role user
- [x] created new user
- [x] disabled user
- [x] created new group

settings
- [x] language settings
- [x] update logo админ
- [x] new password


Remarks:

**Comment by ScharfViktor** [comment]
@micbar QA phase of the rolling release is complete. everything is working successfully

**Comment by micbar** [comment]
# 🎉  Infinite Scale 6.4.0

## Docker

`docker pull owncloud/ocis-rolling:latest` 

OR

`docker pull owncloud/ocis-rolling:6.4.0`

## Binaries

https://download.owncloud.com/ocis/ocis/rolling/6.4.0/

## Changelog

### ocis 
* Bugfix - Set capability response `disable_self_password_change` correctly: [#9853](https://github.com/owncloud/ocis/pull/9853)
* Bugfix - Activity Translations: [#9856](https://github.com/owncloud/ocis/pull/9856)
* Bugfix - The user attributes `userType` and `memberOf` are readonly: [#9867](https://github.com/owncloud/ocis/pull/9867)
* Bugfix - Use key to get specific trash item: [#9879](https://github.com/owncloud/ocis/pull/9879)
* Bugfix - Fix response code when upload a file over locked: [#9894](https://github.com/owncloud/ocis/pull/9894)
* Bugfix - List OCM permissions as graph drive item permissions: [#9905](https://github.com/owncloud/ocis/pull/9905)
* Bugfix - Fix listing ocm shares: [#9925](https://github.com/owncloud/ocis/pull/9925)
* Bugfix - Allow update of ocm shares: [#9980](https://github.com/owncloud/ocis/pull/9980)
* Change - Remove store service: [#9890](https://github.com/owncloud/ocis/pull/9890)
* Enhancement - We now set the configured protocol transport for service metadata: [#9490](https://github.com/owncloud/ocis/pull/9490)
* Enhancement - Microsoft Office365 and Office Online support: [#9686](https://github.com/owncloud/ocis/pull/9686)
* Enhancement - Unified Roles Management: [#9727](https://github.com/owncloud/ocis/pull/9727)
* Enhancement - Added a new role space editor without versions: [#9880](https://github.com/owncloud/ocis/pull/9880)
* Enhancement - Improve revisions purge: [#9891](https://github.com/owncloud/ocis/pull/9891)
* Enhancement - Allow setting default locale of activitylog: [#9892](https://github.com/owncloud/ocis/pull/9892)
* Enhancement - Graph translation path: [#9902](https://github.com/owncloud/ocis/pull/9902)
* Enhancement - Added a new roles viewer/editor with ListGrants: [#9943](https://github.com/owncloud/ocis/pull/9943)
* Enhancement - Handle OCM invite generated event: [#9966](https://github.com/owncloud/ocis/pull/9966)
* Enhancement - Update web to v10.2.0: [#9988](https://github.com/owncloud/ocis/pull/9988)
* Enhancement - Allow blob as connect-src in default CSP: [#9993](https://github.com/owncloud/ocis/pull/9993)
* Enhancement - Bump reva to v2.24.1: [#10028](https://github.com/owncloud/ocis/pull/10028)

### Reva

  *   Bugfix [cs3org/reva#4843](https://github.com/cs3org/reva/pull/4843): Allow update of ocm shares
  *   Bugfix [cs3org/reva#4820](https://github.com/cs3org/reva/pull/4820): Fix response code when upload a file over locked
  *   Bugfix [cs3org/reva#4837](https://github.com/cs3org/reva/pull/4837): Fix OCM userid encoding
  *   Bugfix [cs3org/reva#4823](https://github.com/cs3org/reva/pull/4823): Return etag for ocm shares
  *   Bugfix [cs3org/reva#4822](https://github.com/cs3org/reva/pull/4822): Allow listing directory trash items by key
  *   Enhancement [cs3org/reva#4816](https://github.com/cs3org/reva/pull/4816): Ignore resharing requests
  *   Enhancement [cs3org/reva#4817](https://github.com/cs3org/reva/pull/4817): Added a new role space editor without versions
  *   Enhancement [cs3org/reva#4829](https://github.com/cs3org/reva/pull/4829): Added a new roles viewer/editor with ListGrants
  *   Enhancement [cs3org/reva#4828](https://github.com/cs3org/reva/pull/4828): New event: UserSignedIn
  *   Enhancement [cs3org/reva#4836](https://github.com/cs3org/reva/pull/4836): Publish an event when an OCM invite is generated

### Web

  * Bugfix [owncloud/web#11512](https://github.com/owncloud/web/pull/11512): OCM invite generation body format
  * Bugfix [owncloud/web#11526](https://github.com/owncloud/web/pull/11526): Logout on access token renewal failure
  * Enhancement [owncloud/web#11377](https://github.com/owncloud/web/pull/11377): Replace custom datepicker with native html element
  * Enhancement [owncloud/web#11387](https://github.com/owncloud/web/pull/11387): Display disabled role permissions
  * Enhancement [owncloud/web#11394](https://github.com/owncloud/web/pull/11394): Mark external shares
  * Enhancement [owncloud/web#11484](https://github.com/owncloud/web/pull/11484): Hide versions panel with insufficient permissions
  * Enhancement [owncloud/web#11502](https://github.com/owncloud/web/pull/11502): Support a tags in actions
  * Enhancement [owncloud/web#11508](https://github.com/owncloud/web/pull/11508): Improve tiles view performance
  * Enhancement [owncloud/web#11515](https://github.com/owncloud/web/pull/11515): Add default actions extension point
  * Enhancement [owncloud/web#11518](https://github.com/owncloud/web/pull/11518): Add select all checkbox to tiles view
